apps/cms/emails.py

```diff
@@ -0,0 +1,39 @@
+import csv
+import io
+from email.mime.text import MIMEText
+
+from django.utils import timezone
+from django.utils.translation import ugettext as _
+
+from adhocracy4 import emails as a4_emails
+
+
+class FormEmail(a4_emails.Email):
+    template_name = 'meinberlin_cms/emails/form_submission'
+
+    def get_receivers(self):
+        return [x.strip() for x in self.object.to_address.split(',')]
+
+
+class CsvFormEmail(FormEmail):
+
+    def _generate_csv(self):
+        stream = io.StringIO()
+        fw = csv.writer(stream, delimiter=';', quoting=csv.QUOTE_MINIMAL)
```

Comment:
We need to change the delimiter, as its comma the default.
The default for quoting is already MINIMAL, so we might remove this.